sencha-workspace/SlateTasksStudent/app/view/TaskTree.js

```diff
@@ -216,16 +216,33 @@ Ext.define('SlateTasksStudent.view.TaskTree', {
         }
     },
 
+    afterTasksLoad: function() {
+        var tree = this,
+            sublists = tree.el.select('.slate-tasktree-sublist');
+
+        sublists.each(function(sublist) {
+            sublist.set({
+                'data-natural-height': sublist.getHeight(),
+                style: {
+                    height: 0
+                }
+            });
+        });
+    },
+
     onTreeClick: function(ev, t) {
         var target = Ext.get(t),
-            parentEl,recordId;
-
-        if (target.is('.slate-tasktree-nub.is-clickable')) {
-            target.up('.slate-tasktree-item').toggleCls('is-expanded');
+            treeItem = target.up('.slate-tasktree-item'),
+            sublist = treeItem.down('.slate-tasktree-sublist'),
+            recordId;
+
+        if (target.is('.slate-tasktree-nub.is-clickable') && treeItem && sublist) {
+            var newHeight = treeItem.hasCls('is-expanded') ? 0 : sublist.getAttribute('data-natural-height');
```

Comment:
@ryon don't use the `var` keyword within non-function blocks -- they don't create a variable scope. The `newHeight` variable should be declared above and only set here, like recordId